Refresh: CLIP Part One: Outfits and Clothing

rye-rice opened the pull request:
## About The Pull Request

![Screenshot_11203](https://github.com/shiptest-ss13/Shiptest/assets/58402542/e22d3bdc-acb4-488d-bf4c-74da9de8fe7c)

No, she doesn't read your faxes.

Part 1. Adds clothing and outfits. Doesn't add the species variants as i just want to get this finished my god

Removes the asclepius and corvus. No hard feeling towards the asclepius, it would break with all the changes and so will the corvus. Except hard feelings towards corvus.

The guns will be in another PR pending a gun reorganization

Mass renames minutemen, cmm, to CLIP minutemen, CLIP, and similar

Something actually player facing: the camo fatigues and military webbings have been resprited

## Why It's Good For The Game

![image](https://github.com/shiptest-ss13/Shiptest/assets/58402542/8f4be9f3-2c75-411b-a30b-7709c751e655)

![image](https://github.com/shiptest-ss13/Shiptest/assets/58402542/c330609c-a62e-4c27-a37f-7775ecaec526)


![Screenshot_11204](https://github.com/shiptest-ss13/Shiptest/assets/58402542/8d565b17-5de3-4812-a4a1-97dd1e1d6cbd)


## Changelog
:cl: ryerice, meemofcourse 
add: redraws military webbing and camo fatigues, gives webbings recolor options
add: combat boots and gloves have been given happier colors

add: A ton of things related to the confederated league have been resprited and added
del: Corvus and Asclepius
tweak: CLIP ERTs have been adjusted to be consistent
/:cl:

rye-rice committed:
FUCK this ship, i hope more are cooked up post-refresh

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into re-cmm

rye-rice committed:
repath

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into re-cmm

rye-rice committed:
need to pause

rye-rice committed:
stuff

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into re-cmm

rye-rice committed:
clothinggggggg

rye-rice committed:
gruh

rye-rice committed:
merge

rye-rice committed:
progreesss

rye-rice committed:
poggers, left on at engi outfit

rye-rice committed:
wga

rye-rice committed:
gwa

rye-rice committed:
im lazy

rye-rice committed:
awawawa

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into re-cmm

rye-rice committed:
why the hell is this here

rye-rice committed:
gwa

meemofcourse commented on code/datums/ert.dm:
```diff
@@ -140,46 +140,55 @@
-	rename_team = "Generic Minutemen Team"
-	polldesc = "a Minutemen emergency team"
+	mission = "Resolve the conflict at hand"
+	rename_team = "Quick and Simple CLIP Minutemen Team"
```
rename_team gets shown as the name of the team at the end of the round! i'd recommend you keep it to Generic CLIP Minutemen Team or something along those lines

meemofcourse commented on code/modules/clothing/outfits/ert/minutemen_ert.dm:
it would be useful if you kept the ERT base outfit and internal naming so there's still a clear distinction between standard outfits and ERT outfits. (please make my job easier)

meemofcourse commented on code/modules/vending/wardrobes.dm:
```diff
@@ -541,3 +541,56 @@
+
+/obj/machinery/vending/wardrobe/clip_wardrobe
+	name = "\improper CLIPDrobe"
+	desc = "A very special vendding machine that somehow vends every piece of clothing used by the Cofederated League! Wow! You get the feeling this is meant for debugging"
```
```suggestion
	desc = "A very special vending machine that somehow vends every piece of clothing used by the Confederated League! Wow! You get the feeling this is meant for debugging."
```

meemofcourse commented on code/modules/projectiles/guns/ballistic/smg.dm:
```diff
@@ -258,6 +258,22 @@
 
+/obj/item/gun/ballistic/automatic/smg/cm5/compact
+	name = "\improper CM-5c"
+	desc = "The compact conversion of the CM-5. While not exactly restricted, it is looked down upon due to CLIP's doctrine on medium-longrange combat, however it excells at close range and is, well, compact. You also feel like this gun is unfinished somewhat. Chambered in 9mm."
```
```suggestion
	desc = "The compact conversion of the CM-5. While not exactly restricted, it is looked down upon due to CLIP's doctrine on medium-longrange combat, however it excels at close range and is very lightweight. You feel like this gun is mildly unfinished. Chambered in 9mm."
```

meemofcourse commented on code/modules/clothing/outfits/factions/minutemen.dm:
```diff
@@ -48,42 +101,49 @@
 	backpack = /obj/item/storage/backpack
 	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1, /obj/item/gun/ballistic/revolver/mateba=1)
 
+/datum/outfit/job/clip/minutemen/captain/general/admiral // for flavor, might remove outright
```
might wanna add a post_equip that updates the ID card's name to admiral

rye-rice replied:
i have no idea how to do this

meemofcourse commented on code/modules/clothing/outfits/factions/minutemen.dm:
```diff
@@ -311,42 +431,237 @@
+
+//grunts - for erts as well
+
+/obj/item/twenty_pounds_of_ice
```
thanks rye

rye-rice committed:
wa

rye-rice committed:
Update code/modules/vending/wardrobes.dm

Co-authored-by: meem <75212565+meemofcourse@users.noreply.github.com>
Signed-off-by: retlaw34 <58402542+retlaw34@users.noreply.github.com>

rye-rice committed:
Update code/modules/projectiles/guns/ballistic/smg.dm

Co-authored-by: meem <75212565+meemofcourse@users.noreply.github.com>
Signed-off-by: retlaw34 <58402542+retlaw34@users.noreply.github.com>

rye-rice committed:
auhg

meemofcourse committed:
you should merge this pr without looking (#7)

rye-rice committed:
Merge branch 'master' of https://github.com/shiptest-ss13/Shiptest into re-cmm

meemofcourse committed:
Merge branch 'master' into re-cmm

meemofcourse committed:
vela fixes (#8)

* you should merge this pr without looking

* tries to fix up the vela

kill people with a ship covered in moths that explodes and the moths go everywhere

* nooo mech techs!!

noooooooooooooo mech techs!!!!!

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into pr/2783

meemofcourse committed:
oops

meemofcourse committed:
Update minutemen_vela.dmm

meemofcourse committed:
fix linters

meemofcourse committed:
kill people with a car covered in hammers that explodes and the hammers go everywhere

meemofcourse committed:
BARD ERT Update

meemofcourse committed:
oops

rye-rice committed:
Merge branch 're-cmm' of https://github.com/retlaw34/Shiptest into re-cmm

rye-rice committed:
forgor 1

Imaginos16 reviewed (approved):
Sprites are good, good luck with everything else in the PR!

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into pr/2783

meemofcourse committed:
fix conflicts

meemofcourse committed:
Update inteq_vaquero.dmm
